### PR TITLE
Randomized the jobs on the joblist

### LINKED ISSUE #566: Company: Randomize job order on joblist

### PR DESCRIPTION
**Pull request by wsinnema**
Fixes #566

### CODE REVIEW

File: module/Company/view/company/company/job-list.phtml [hunk 1]
```diff
@@ -59,6 +59,8 @@ $this->headTitle($this->translate('Jobs'));
         <?php
 
 
+        // Randomize the order of companies
+        shuffle($jobList);
```
Comment (jszanto) [inline module/Company/view/company/company/job-list.phtml]: Doing this, as I mentioned earlier, also randomizes the order on the page which shows all jobs for a company. This is very anoying in my opinion when trying to browse through all the jobs for a specific company.

Comment (wsinnema) [inline module/Company/view/company/company/job-list.phtml]: I didn't notice, now it is fixed

File: module/Company/view/company/company/job-list.phtml [hunk 2]
```diff
@@ -59,6 +59,10 @@ $this->headTitle($this->translate('Jobs'));
         <?php
 
 
+        // Randomize the order of companies
+        if ($randomize):
+            shuffle($jobList);
+        endif
```
Comment (kokx) [inline module/Company/view/company/company/job-list.phtml]: Within view code, we have two styles of using control statements like `if`:

```php
<?php if (/* condition*/): ?>
HTML
<?php endif ?>
```

And:

```php
<?php
if (/* condition */) {
    /* PHP code */
}
```

Mixing them only makes code less readable.

Comment (wsinnema) [inline module/Company/view/company/company/job-list.phtml]: It's fixed now

**Comment by kokx** [comment]
Why is this closed? Shouldn't it be merged?

**Comment by wsinnema** [comment]
Because the request listed commits that are already merged. I opened the same request again with the correct commits

**Pull request closed without merging**